adagrams/game.py

```diff
@@ -1,11 +1,193 @@
+import random
+from collections import Counter
+
+#creat letter pool 
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+
+
 def draw_letters():
-    pass
+    #make a copy of the letter pool
+    user_pool = dict(LETTER_POOL)
+
+    #create a list to sotre user drawn on hand
+    hand = []
+    
+    #use while loop to draw a random letter from letter pool
+    while len(hand) < 10:
+        # get all the letters from letter pool and store in a list
+        letters = list(LETTER_POOL.keys())
```

Comment:
`letters` has 26 elements, the letters A-Z. However, `letters` should contain 98 elements. Check out the distribution of letters [here](https://github.com/Ada-c22/adagrams-py?tab=readme-ov-file#distribution-of-letters). Therefore the list should have 9 A's, 2 B's, etc like `["A", "A", "A", "A", "A", "A", "A", "A", "A", "B", "B", etc]`

How can you use `LETTER_POOL` to create such a list? _Please reply to this comment to let me know._ 

Thank you!